src/gui/tray/usermodel.cpp

```diff
@@ -945,6 +1026,89 @@ void User::forceSyncNow() const
     FolderMan::instance()->forceSyncForFolder(getFolder());
 }
 
+void User::slotCapabilitiesChanged()
```

Comment:
if this is only related to group folders, maybe the name should reflect it

Reply:
> if this is only related to group folders, maybe the name should reflect it

@mgallien this is, now, but then we may want to add more code there so the specific name won't make sense anymore, at least that's the review comment that I've been receiving from you in my last PRs